src/views/ErrorBoundary/PageNotFound.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { useTracking } from 'react-tracking';
+
+import TextInput from '../../components/atomic/TextInput';
+import { useStateValue } from '../../store/store';
+import { i18n } from '../../utils';
+
+const PageNotFound = () => {
+	const [{ canonicalHost, language }] = useStateValue();
+	const [searchText, updateSearchText] = useState('');
+	const tracking = useTracking();
+
+	useEffect(() => {
+		const pageTitle = i18n.pageNotFoundTitle[language];
+		tracking.trackEvent({
+			event: 'SitewideSearchApp:Load:PageNotFound',
```

Comment:
Fix this please. It should be `DrugDictionaryApp`

Reply:
Same as above.